ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+resources:
+  '*':
+    limits:
+      memory: 4Gi
```

Comment:
(optional)isn't it a bit much for a build?

Reply:
turns out this limit is actually necessary
```
  * could not run steps: step build failed: test "build" failed: the pod ci-op-5gi2xhwr/build failed after 40s (failed containers: test): ContainerFailed one or more containers exited
Container test exited with code 2, reason OOMKilled
```